index.html

```diff
@@ -34,6 +34,187 @@ <h2>I'm a Software Developer</h2>
             <li><a href="https://twitter.com/anyarsencarta"><img src="./logos/Vector-M.png" alt="Linkedin"></a></li>
           </ul>
         </section>
+
+        <section class="extra">
+          <section class="works-centered">
+            <h2 class="works-title">My Recent Works</h2>
+            <hr>
+            <img src="./logos/My Recent Works.png" alt="My Recent Works">
+            <h2 class="multi-stories">Multi-Post Stories</h2>
+            <p>
+              A daily selection of privately personalized reads; no accounts or sign-ups 
+              required. has been the industry's standard dummy text ever since the 1500s, 
+              when an unknown printer took a standard dummy text.
+            </p>
+            <ul class="project-list">
+              <li><a class="start-list" href="#">HTML</a></li>
+              <li><a class="second-list" href="#">CSS</a></li>
+              <li><a class="third-list" href="#">Javascript</a></li>
+            </ul>
+            <button>See Project</button>
+          </section>
+
+          <section class="print-data">
```

Comment:
- **[OPTIONAL]** Please 🙏🏽 for each card we could use the article tag instead of the section tag for the best practices of HTML & CSS.